cardano-api/gen/Test/Gen/Cardano/Api/Typed.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE GADTs #-}
+{-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE NamedFieldPuns #-}
```

Comment:
`Warning in module Test.Gen.Cardano.Api.Typed: Unused LANGUAGE pragma ▫︎ Found: "{-# LANGUAGE LambdaCase #-}"`